intro-to-git/intro-to-git-vocab-and-concepts.md

```diff
@@ -0,0 +1,127 @@
+# Intro to Git: Vocab and Concepts
+
+## Learning Goals
+
+- Explain how Git is version control system used to version code
+- Define a repository in Git
+- Compare a Git repository to a project
+- Define a commit in Git
+
+## Introduction
+
+Recall that _versions_ in software are different states of a codebase.
+
+Consider how the version of you from 10 years ago is different from the version of you from 10 minutes ago (which is different from the you of right now). Are those three versions of you very different from each other? Between the you from 10 years ago and the present you, **what was lost? What was gained? And what was modified?**
+
+Between the you from 10 minutes ago and the present you, what was lost, gained, or modified? The difference between those two versions of you could be huge, or they could be small, incremental, barely noticeable changes.
+
+What a codebase looked like at one point in time could be a different version of the code right now. We can say that there is a version of a codebase that passed three tests, and a version of the same codebase that passed four tests.
+
+How can the idea of versioning help us build software?
+
+## Vocabulary and Synonyms
+
+| Vocab | Definition | Synonyms | How to Use in a Sentence |
+| ----- | ---------- | -------- | ------------------------ |
+
+Version
+Version Control System
+Git
+Repository
+Tracking (in Git)
+Commit
```

Comment:
```suggestion
| Vocab | Definition | Synonyms | How to Use in a Sentence |
| ----- | ---------- | -------- | ------------------------ |
| Version | Code at a specific point in time labeled with a number based identifier | draft, copy | "I created a separate version of this file so it wouldn't overwrite the previous version."
| Version Control System | A tool that records changes to a file or set of files over time as versions | VCS | "As a project grows, it's wise to use a VCS to keep track of all its changes and progressions.","Good thing the VCS kept track of my previous commits, otherwise, I would have lost all that work!" |
| Git | A distributed version control system that makes snapshots of files or multiple files in a repository to be later referenced as a version. | - | "The problem with *git* jokes is, everyone has their own version" | 
| Repository | A directory that stores project files | Repo, folder, directory | "The project is in a local repository on my machine." |
Commit | An individual change to file(s), identified with a hash | - | "The recent commit fixed the bug in our project" | 
```

Reply:
I might push back on your synonyms of "version"; I get what you're saying, but it just so happens that a definition of version that was in the context of releases has been given

Also LOL!!!!! Your Git joke LOLL